.gitignore

```diff
@@ -13,3 +13,9 @@
 # ignore default modules from composer
 /assets/
 /resources/
+
+# ignore default theme
+/themes/simple/
+
+# ignore platform requirements
+/silverstripe-dynamodb/
```

Comment:
We should really just make this a vendor module

Reply:
Looks like it actually is now... might just need an update in this project. Will give it a go.